src/ol_infrastructure/applications/edxapp/k8s_resources.py

```diff
@@ -404,6 +607,21 @@ def create_k8s_resources(
             mount_path="/openedx/config",
         )
     )
+
+    # Finally, actually define the CMS deployment
+    # Start with a pre-deployment job that runs the migrations
+    cms_pre_deploy_migrate_job = _create_pre_deploy_job(
```

Comment:
We will want to make sure that the LMS migrations _always_ run before the CMS migrations

Reply:
Ohhhhh, okay I didn't know that. Hm. Okay. I can make that happen I think.